sys/include/evtimer.h

```diff
@@ -15,24 +15,20 @@
  *
  * @note    Experimental and likely to replaced with unified timer API
  *
- * RIOT's main timer subsystem is @ref sys_xtimer "xtimer", but for many
- * applications @ref sys_xtimer "xtimer's" 64-bit absolute time values are
+ * RIOT's main timer subsystem is @ref sys_ztimer "ztimer", but for many
+ * applications @ref sys_ztimer "ztimer's" 64-bit absolute time values are
```

Comment:
The 64 bit is no longer true. However, each ztimer still has a function pointer for the callback. When a number of timeouts with the same callback function need to be scheduled, evtimer is using less RAM (due to storing the callback function only once).

Reply:
I rewrote the comparison with ztimer a bit.